README.md

```diff
@@ -122,7 +122,8 @@ Here's a list of the currently-support config items.  These can go in your `/con
                     'key' : session.user.id,
                     'name' : session.user.fullname,
                     'email' : session.user.email,
-                    'country' : session.user.country
+                    'country' : session.user.country,
+                    'privateAttributes' : ['email']
```

Comment:
I thought it would be useful to include the privateAttributes key in the example here, but feel free to remove if it seems overly complicated for this example